shell/plugins/steve/subscribe.js

```diff
@@ -456,41 +451,54 @@ export const actions = {
   }, e) {
     clearTimeout(state.queueTimer);
     state.queueTimer = null;
-    if (e.type === EVENT_DISCONNECT_ERROR) {
-      // determine if websocket notifications are disabled
-      const perfSetting = rootGetters['management/byId'](MANAGEMENT.SETTING, SETTING.UI_PERFORMANCE);
-      let disableGrowl = false;
+    const dateFormat = escapeHtml( rootGetters['prefs/get'](DATE_FORMAT));
+    const timeFormat = escapeHtml( rootGetters['prefs/get'](TIME_FORMAT));
+    const time = e?.srcElement?.disconnectedAt || Date.now();
 
-      if ( perfSetting?.value ) {
-        disableGrowl = JSON.parse(perfSetting.value).disableWebsocketNotification || false;
-      }
+    const timeFormatted = `${ day(time).format(`${ dateFormat } ${ timeFormat }`) }`;
+    const url = e?.srcElement?.url;
+    const tries = state?.socket?.tries;
 
-      if ( !disableGrowl ) {
-        // do not send a growl notification unless the socket stays disconnected for more than MINIMUM_TIME_DISCONNECTED
-        setTimeout(() => {
-          if (state.socket.isConnected()) {
-            return;
-          }
-          const dateFormat = escapeHtml( rootGetters['prefs/get'](DATE_FORMAT));
-          const timeFormat = escapeHtml( rootGetters['prefs/get'](TIME_FORMAT));
-          const time = e?.srcElement?.disconnectedAt || Date.now();
-
-          const timeFormatted = `${ day(time).format(`${ dateFormat } ${ timeFormat }`) }`;
-          const url = e?.srcElement?.url;
-
-          const t = rootGetters['i18n/t'];
-
-          dispatch('growl/error', {
-            title:         t('growl.disconnected.title'),
-            message:       t('growl.disconnected.message', { url, time: timeFormatted }, { raw: true }),
-            icon:          'error',
-            earliestClose: time + MINIMUM_TIME_NOTIFIED + MINIMUM_TIME_DISCONNECTED,
-            url
-          }, { root: true });
-        }, MINIMUM_TIME_DISCONNECTED);
+    const t = rootGetters['i18n/t'];
+
+    // determine if websocket notifications are disabled
+    const perfSetting = rootGetters['management/byId'](MANAGEMENT.SETTING, SETTING.UI_PERFORMANCE);
+    let disableGrowl = false;
```

Comment:
This setting seems to be ignored - we still want to disable growls via this setting, just in case there is still an issue.

Reply:
it was being used on EVENT_DISCONNECT_ERROR error but not EVENT_CONNECT_ERROR which has since been corrected and pushed. Thanks for catching that!